makeflow/src/mf-mesos-executor

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# This script uses MESOS_SOURCE_DIR and MESOS_BUILD_DIR which come
+# from configuration substitutions.
+MESOS_SOURCE_DIR=/afs/nd.edu/user37/ccl/software/external/mesos-0.26.0/src/mesos-0.26.0_build
```

Comment:
This isn't going to work outside of ND.  Do we need to have some kind of environment var or makeflow option to propagate the necessary info?

@btovar any suggestions on how to handle this?